src/Client/ContentHubServiceVersion1.php

```diff
@@ -56,7 +56,14 @@ public static function new(): ContentHubServiceInterface {
    * {@inheritDoc}
    */
   public function getClients(): array {
-    return \Drupal::service('acquia_contenthub.acquia_contenthub_subscription')->getClients();
+    $clients = [];
+    foreach (\Drupal::service('acquia_contenthub.acquia_contenthub_subscription')->getSettings()->getWebhooks() as $webhook) {
```

Comment:
Code looks good to me from 1.x perspective but we should use DrupalServiceFactory to get the service instead of calling it directly from container but since container is already being used in rest of the methods in this class, I guess it should be Ok for now.
But this needs to be functionally tested once.

Reply:
Makes sense, but I did as minimal changes as possible to allow it to work. If we decide to change to DrupalServiceFactory then we would have to do the same in `ContentHubServiceVersion2` also to keep consistency. I also tested this manually during the upgrade from 1.x to 2.x using D9. If someone else can test this piece would be great.

Reply:
Yep I agree, that's why I said we can merge the PR as it is once it's functionally tested. Will do that tomorrow morning and merge it once it's approved and tested.